### ISSUE TITLE
column name duplicated in allgensnp

### ISSUE THREAD

**Issue by sinarueeger** [ropensci/rsnps#67]
Column name `name` is duplicated in `allgensnp`:

``` r

dat <- rsnps::allgensnp("rs7412")
head(dat)
#>     name chromosome position                name   id genotype_id
#> 1 rs7412         19 44908822        R.M. Holston   22           8
#> 2 rs7412         19 44908822 Charles G. Sullivan 5326        3834
#> 3 rs7412         19 44908822   Glenn Allen Nolen   19           7
#> 4 rs7412         19 44908822        Angel Harris  495         223
#> 5 rs7412         19 44908822          Dan Bolser  254          NA
#> 6 rs7412         19 44908822           Mom to AG  387         173
#>   local_genotype
#> 1             CC
#> 2             CC
#> 3             CC
#> 4             CC
#> 5           <NA>
#> 6             CC
```

<sup>Created on 2019-05-07 by the [reprex package](https://reprex.tidyverse.org) (v0.2.1)</sup>

This makes it difficult to query the data.frame later.

**Comment by jooolia** [comment]
Still relevant.

**Comment by jooolia** [comment]
Fixed in #156
